requirements.txt

```diff
@@ -19,8 +19,8 @@ enum34==1.1.10            # via google-cloud-tasks, grpcio
 funcsigs==1.0.2           # via mock
 future==0.18.2            # via -r requirements.in
 futures==3.3.0            # via google-api-core, grpcio
-google-api-core[grpc]==1.22.2  # via google-cloud-tasks
-google-auth==1.21.1       # via google-api-core
+google-api-core[grpc]==1.22.4  # via google-cloud-tasks
```

Comment:
Just to check -- what was the context behind these changes / why are they in this PR?

Reply:
@seanlip I'm not sure - I've been having a lot of issues with the requirements.txt file since this past Saturday. For example, when I tried to push these changes to git a couple days ago, I got an error message saying rerun the scripts.install_third_party script and then when I ran the scripts.install_third_party it failed saying run the scripts.regenerate_requirements script....worse on Saturday nothing would work for a bit (scripts.start, running tests)...because for some reason in the install_backend_python_libs file the subprocess.check_call for running 
        ['python', '-m', 'scripts.regenerate_requirements'] (line 575 in install_backend_python_libs) kept hanging even though running scripts.regenerate_requirements worked when I ran it by itself.....then that problem went away the next day (though I've still had issues) I'm not sure what's going on

Reply:
@seanlip since requirements.txt is generated, won't it automatically be updated when downstream dependencies change their dependencies? Since these are inferred versions, I was assuming it's just an unrelated change that will crop up.

Reply:
Sorry, yes it was automatically generated for me

Reply:
Yeah I wish I could've included a stack trace but there wasn't anything at all when install_backend_python_libs was hanging...it just hung. But yes, I'll copy and paste the stack trace for when I update others about errors in the future :)

Reply:
Sorry that's what I meant @seanlip. This is an inherited dependency, so we never specify the version. I'm not actually sure how we can solve that, but it's worth discussing further outside this PR.